test/e2e/handler/utils.go

```diff
@@ -165,6 +165,9 @@ func deletePolicy(name string) {
 	policy := &nmstatev1alpha1.NodeNetworkConfigurationPolicy{}
 	policy.Name = name
 	err := framework.Global.Client.Delete(context.TODO(), policy)
+	if apierrors.IsNotFound(err) {
```

Comment:
I know it's a corner case, but don't we want to be on the safe side and verify also that NodeNetworkConfigurationEnactment was deleted although policy not found?

Reply:
That's super good question, we know we are missing adding the block owner deletetion flag [1] at [2] but we will do that at different PR, I think

[1] ownerReference.blockOwnerDeletion=true-> https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/#foreground-cascading-deletion

[2] https://github.com/nmstate/kubernetes-nmstate/blob/fad730940ba634da99785b3dc5d99caf696c70af/pkg/apis/nmstate/v1alpha1/nodenetworkconfigurationenactment_types.go#L76

Reply:
@yboaron I have create a PR for that https://github.com/nmstate/kubernetes-nmstate/pull/526